### PR TITLE
Adding full code

### PR DESCRIPTION
**Pull request by mucalo**
Hey, Nate,

As agreed, this should add the full code to the repo.
Please double check the links in Readme.md - I don't know what the final URL is going to be.
Also, I was not sure what @okta/okta-angular 1.0.0 is officially called, so I:
- called it Okta Angular SDK
- linked to the npm package page

Please change this if needed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+## Links
+
+This example uses the following libraries provided by Okta:
+
+* [Okta Angular SDK](https://www.npmjs.com/package/@okta/okta-angular)
+* [Okta .NET SDK](https://github.com/okta/okta-sdk-dotnet)
+
+## Help
+
+Please post any questions as comments on the [blog post](URL???), or visit the [Okta Developer Forums](https://devforum.okta.com/). You can also email developers@okta.com if you would like to create a support ticket.
```
Comment (nbarbettini) [inline README.md]: I'll add this URL 👍

File: README.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    options.Audience = "api://default";
+});
+```
+
+#### Client configuration
+
+Set the `issuer` and copy the `clientId` of the Okta application into `ClientApp/router.js`.
+
+```javascript
+Vue.use(Auth, {
```
Comment (nbarbettini) [inline README.md]: These instructions need to be updated for Angular.

File: Backend/Backend/Controllers/ValuesController.cs [hunk 3]
```diff
@@ -0,0 +1,44 @@
+﻿using System;
```
Comment (nbarbettini) [inline Backend/Backend/Controllers/ValuesController.cs]: Can you remove this file @mucalo? IIRC it is not used in the tutorial.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+
+dotnet run
+```
+
+## Links
+
+This example uses the following libraries provided by Okta:
+
+* [Okta Angular SDK](https://www.npmjs.com/package/@okta/okta-angular)
+* [Okta .NET SDK](https://github.com/okta/okta-sdk-dotnet)
```
Comment (nbarbettini) [inline README.md]: I don't think you're using `Okta.Sdk` in this project. Can you confirm @mucalo?

**Comment by mucalo** [comment]
Hey, @nbarbettini 
I have pushed the following changes to master:

- deleted the ValuesController.cs as it has not been used
- removed the usage of Okta Core Client 0.33 as it has not been used
- updated the Readme.md to remove Vue.js references and substitute with Angular references
- updated the Readme.md to update installation instructions - can you check on these please?
- updated the Readme.md to update the libraries used

Please let me know if you need anything else at this point.

**Review by nbarbettini — APPROVED** [review]
Looks good, thanks!

**Pull request merged**